tests/test_lib_token.py

```diff
@@ -1934,6 +1934,7 @@ def test_01_successfully_change_pin(self):
                                                   "g": g})
         self.assertFalse(r)
         self.assertEqual("Please enter a new PIN", reply_dict.get("message"))
+        self.assertEqual("generic_pin_reset", reply_dict.get('multi_challenge')[0].get('challenge_type'))
```

Comment:
We should also test somewhere, that the `standard` challenge type is returned if there is no PIN-reset or resync-challenge.

Reply:
7d835b180677b94d0de6d320c9d8841caff697a1